src/backend/routes/me.py

```diff
@@ -55,6 +56,9 @@
     access_token_expires = timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
     access_token = create_access_token(
         data={"login": user.email}, expires_delta=access_token_expires)
+    user.last_signin_date = datetime.now()
```

Comment:
Приведём всё к одной временной зоне.
```suggestion
    user.last_signin_date = datetime.utcnow()
```